### ISSUE TITLE
[BUG] Negative loss/stabilize-train and loss/val

### ISSUE THREAD

**Issue by orcinus** [victorchall/EveryDream2trainer#119]
For some reason, i keep getting negative loss/stabilize-train and loss/val:

![image](https://user-images.githubusercontent.com/1574258/227688899-f5386dcc-e9d4-44b7-ab13-721b68abd23a.png)

It starts off at 0, and goes into negative value each epoch.
I'm not sure why.

I'm training on 1065 images, 10% split off for validation.
Stabilize split proportion set to 0.10 too.
1 batch.

**Comment by victorchall** [comment]
Damian set it up to run negative relative to the pre-training val/loss pass. 

I just pushed an option to validation config json so you can flip it either way.  

https://github.com/victorchall/EveryDream2trainer/blob/main/validation_default.json#L21

Default as delivered in validation_defeault.json is false, which is normal positive values, true would give you negative number, how Damian configured it before and what you are seeing above.

**Comment by orcinus** [comment]
Aaaah! Gotcha.
Thanks!
